frontend/src/components/Paper/PaperCard/PaperCard.js

```diff
@@ -46,9 +47,9 @@ class PaperCard extends Component {
     ).slice(0, 10).filter(
         (keyword) => keyword.type === type,
     ).map(
-        (keyword) => (
+        (keyword, index) => (
```

Comment:
Well, I think we can make unique keys by combining id of the paper and source of the paper.(ex> recommendation_1) To do this, we will need to pass the source prop.